core/src/modules/cloudCli.ts

```diff
@@ -74,7 +74,7 @@ export namespace CloudCLI {
       );
 
       let lastState: string;
-      while (true) {
+      for (;;) {
```

Comment:
Fix for [no-constant-condition](https://eslint.org/docs/rules/no-constant-condition)

Reply:
Do you think we should disable that rule? We could rewrite this function to not need an infinite loop but I'm not it's worth the refactor time.